### PR TITLE
Posteventfist

### PR DESCRIPTION
**Pull request by QAI-07**
## About The Pull Request

> Hopefully makes the Fist of Sol fight a bit more enjoyable and less of a bullet sponge.

## Why It's Good For The Game

> Winnable boss fight good.

## Changelog
:cl:
tweak: rebalanced the fist of sol's stats.

add: Wrath of Sol - Special Ops Taskforce

/:cl:

### CODE REVIEW

File: nsv13/code/modules/overmap/ai-skynet.dm [hunk 1]
```diff
@@ -515,6 +515,25 @@ Adding tasks is easy! Just define a datum for it.
 	audio_cues = list()
 	taunts = list("Your assault on Rubicon only served to distract you from the real threat. It's time to end this war in one swift blow.")
 	fleet_trait = FLEET_TRAIT_DEFENSE
+	
+/datum/fleet/syndicate/AbassisWrath
```
Comment (DeltaFire15) [inline nsv13/code/modules/overmap/ai-skynet.dm:519]: Please avoid using capital letters in typepaths.

File: nsv13/code/modules/overmap/types/syndicate.dm [hunk 2]
```diff
@@ -429,6 +429,44 @@
...
+	integrity_failure = 5000
+	speed_limit = 16
+	flak_battery_amount = 3
+	max_tracking_range = 90
+	armor = list("overmap_light" = 99, "overmap_medium" = 65, "overmap_heavy" = 40)
+	ai_controlled = TRUE
+	shots_left = 500
+	missiles = 30
+	ai_behavior = AI_AGRESSIVE
+	ai_flags = AI_FLAG_ELITE | AI_FLAG_BATTLESHIP | AI_FLAG_SWARMER
```
Comment (DeltaFire15) [inline nsv13/code/modules/overmap/types/syndicate.dm:455]: swarmer probably isn't the greatest ai flag for them to have as it overrides behavior and is primarily meant for light vessels. Destroyer and Battleship synergize, on the other hand.

**Comment by Bokkiewokkie** [comment]
Maybe wait for your other PR to get merged first 🤔💭 ?

**Comment by QAI-07** [comment]
I'll see you in four months then.

**Pull request closed without merging**